### PR TITLE
Bump version to 0.18.1, rename '--reporters' CLI parameter back to '--reporter' (close #1881)

### LINKED ISSUE #1881: Reporter JSON being silently ignored.
### Are you requesting a feature or reporting a bug?
Reporting a bug.

### What is the current behavior?
testcafe ignores the --reporter=json option. 

### What is the expected behavior?
To use that built-in module.

### How would you reproduce the current behavior (if this is a bug)?
```sudo /usr/bin/docker run --rm   -e CI=1  -v `pwd`/tests:/tests   --net=dockernet   testcafe/testcafe   'chromium --no-sandbox --kiosk --disable-translate --disable-infobars' --reporter json   /tests/test.js```

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system:
* testcafe version:
* node.js version:

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

File: test/server/cli-argument-parser-test.js [hunk 1]
```diff
@@ -349,5 +350,45 @@ describe('CLI argument parser', function () {
...
+            { long: '--color' },
+            { long: '--no-color' }
+        ];
+
+        var parser  = new CliArgumentParser('');
+        var options = parser.program.options;
+
+        expect(options.length).eql(EXPECTED_OPTIONS.length, WARNING);
+
+        for (var i = 0; i < EXPECTED_OPTIONS.length; i++)
```
Comment (AndreyBelym) [inline test/server/cli-argument-parser-test.js]: maybe
```
expect(options).to.includeDeepMembers(EXPECTED_OPTIONS)
```

Comment (churkin) [reply test/server/cli-argument-parser-test.js]: Looks good :)

Comment (AndreyBelym) [reply test/server/cli-argument-parser-test.js]: We found that it's not useful in this case.

**Pull request merged**